.gitignore

```diff
@@ -2,4 +2,6 @@
 node_modules
 client/dist/bundle.js
 config/auth.js
-.env
\ No newline at end of file
+.env
+package-lock.json
```

Comment:
`package-lock.json` and `package.json` should be included in source control - remove them from your gitignore